src/Mpdf.php

```diff
@@ -12970,12 +12977,18 @@ function SetFooter($Farray = [], $side = '')
 
 	/* -- WATERMARK -- */
 
-	function SetWatermarkText($txt = '', $alpha = -1)
+	function SetWatermarkText($txt = '', $alpha = -1, $size = 120, $pos = null)
 	{
 		if ($alpha >= 0) {
 			$this->watermarkTextAlpha = $alpha;
 		}
 		$this->watermarkText = $txt;
+		$this->watermark_font_size = $size;
+
+		if (!is_array($pos)) {
+			$pos = [0, 0];
```

Comment:
Again, setting an arbitrary value to a config variable from a method call will lead to unexpected behaviour. I know, the SetWatermarkImage does the same, but that's not a reason to encourage this in a new code.